### ISSUE TITLE
spilo not working on kubernetes 1.10

### ISSUE THREAD

**Issue by pitabwire** [zalando/spilo#233]
Spilo 1.4-p7 has stopped working after upgrading to the latest kubernetes version on GKE. This has been traced traced to patroni which has already fixed the issue. Is there a spilo release based on the latest patroni we can use?

**Comment by hjacobs** [comment]
@pitabwire yes, Spilo was also fixed, see this PR for our PostgreSQL Operator which now runs on Kubernetes 1.10 (and uses Spilo): https://github.com/zalando-incubator/postgres-operator/pull/301/files

**Comment by hjacobs** [comment]
Here the existing (new) Spilo images: https://registry.opensource.zalan.do/teams/acid/artifacts/spilo-cdp-10/tags

I don't know which one is correct and recommended for production, I think the maintainers should add a new release linking to the right Docker image: https://github.com/zalando/spilo/releases

**Comment by hjacobs** [comment]
For reference the fix for Patroni was: https://github.com/zalando/patroni/pull/670

**Comment by pitabwire** [comment]
Thanks @hjacobs, are the images https://registry.opensource.zalan.do/v1/repositories/acid/spilo-10/tags exactly the same as the ones here https://registry.opensource.zalan.do/teams/acid/artifacts/spilo-cdp-10/tags?

**Comment by Jan-M** [comment]
This image ```registry.opensource.zalan.do/acid/spilo-cdp-10:1.4-p8``` should work.

The naming has changed for the images that are build automatically.

It is also the new default image for our Postgres operator.

**Comment by hjacobs** [comment]
@pitabwire "CDP" is our internal Continuous Delivery Platform, so I assume that the CDP images are just "newer" --- the maintainers (@CyberDem0n?) should give an authoritative answer.

**Comment by Jan-M** [comment]
I am told we will tag and release a spilo-10 docker image. That one will be added to releases here.

**Comment by CyberDem0n** [comment]
I just pushed the new spilo-10 image: `registry.opensource.zalan.do/acid/spilo-10:1.4-p8`
